easybuild/easyconfigs/c/Control-FREEC/Control-FREEC-11.5-GCC-8.2.0-2.31.1.eb

```
@@ -0,0 +1,30 @@
+# uploaded by J. Sassmannshausen
+
+easyblock = 'MakeCp'
+
+name = 'Control-FREEC'
+version = '11.5'
+
+homepage = 'https://github.com/BoevaLab/FREEC'
+description = """Copy number and genotype annotation from whole 
+genome and whole exome sequencing data."""
+
+toolchain = {'name': 'GCC', 'version': '8.2.0-2.31.1'}
+
+source_urls = ['https://github.com/BoevaLab/FREEC/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['30f24c6db628eb5ce5a5f8ed8e83763915554edb02d940fee2607e0ad7e705a7']
+
+# builddependencies = [('binutils', '2.31.1')]
```

@sassy-crick  please remove this comment?

---

@smoors: I have removed the obsolete build file and the builddependencies, as requested. 
@boegel: thanks for pointing that out to me. Am I right in assuming that this will come into force ones EB-4.x is released or shall I do that from now on?

---

@sassy-crick the 4.x branch has been merged into the develop branch, so you should indeed use the new conventions from now on.

---

@smoors Ok. I will try and bear it in mind. However, in the current case as we are using this toolchain now:

```suggestion
toolchain = {'name': 'GCC', 'version': '8.2.0-2.31.1'}
```
I think I do not need to make any changes here. Is this correct?